### PR TITLE
Feature/cypress

### PR DESCRIPTION
**Pull request by mayakappen**
> Cypress
-This pull request adds in cypress testing functionality
-It gave me more than I asked for but it includes lots of examples for tests within the cypress file
-The tests I wrote are in cypress/e2e/spec.cy.js
-Mine only cover a few end to end tests for when the user visits the webpage, there is no component testing yet.

### CODE REVIEW

**Comment by mayakappen** [comment]
I can also delete these files and push up another without all of the extra stuff, up to you!

File: cypress/e2e/spec.cy.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+describe('/App/App.js', () => {
```
Comment (mayakappen) [inline cypress/e2e/spec.cy.js:1]: These are the tests I wrote: navigating to page, seeing the title, seeing movies

**Review by mayakappen — COMMENTED** [review]
I think I could probably fix this and only send up what we need lol
Let me know if you find this stuff useful though! Maybe to have as a refrence

**Comment by aspitz1** [comment]
It was helpful to look at the example docs, but I think it is a good idea to delete them and push up the branch with just the one we need.

Looks great though, great job on working that out!

**Comment by aspitz1** [comment]
I went ahead and pulled it down, deleted the files, and pushed it up so I could merge it.

**Comment by mayakappen** [comment]
Oh thank you so much for that Anna!

**Pull request merged**